pattern/variable/Variable.java

```diff
@@ -79,21 +79,25 @@ public ThingVariable asThing() {
         throw GraknException.of(INVALID_CASTING, className(this.getClass()), className(ThingVariable.class));
     }
 
-    public void addResolvedType(Label label) {
+    public void addSingleResolvedType(Label label) {
```

Comment:
renamed because before you have to distinguish `addResolvedType` and `addResolvedTypes`, which is a little inconvenient.

Reply:
the pattern to distinguish `singular` vs `plural` method names is actually very common across our codebase, @maxbaxt . We don't like things too verbose especially when a shorter name does the job non-ambiguously. So let's revert this one again.

Reply:
Can you address this @maxbaxt ?

Reply:
Changed it back to `addResolvedType()`